internal/httputil/certpoolwatcher.go

```diff
@@ -66,6 +68,7 @@ func NewCertPoolWatcher(caDir string, log logr.Logger) (*CertPoolWatcher, error)
 		if err := watcher.Add(p); err != nil {
 			return nil, err
 		}
+		logPath(p, log)
```

Comment:
This only logs the contents when the watcher is created, right? It seems like it would be useful to log as events come in and the watched contents change.